ci-operator/config/openshift/external-dns/openshift-external-dns-release-1.1.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  base:
+    name: "4.12"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.18-openshift-4.12:
```

Comment:
golang-1.19 is used for the external-dns-operator, and 1.18 for this?  It would be better to use the same golang version or a comment on the difference.

Reply:
Yes, it would be better. We have less control over the controller code as it's rebased from the upstream. For this release the upstream wasn't updated to 1.19 yet.